UKHomeOffice/nmsw-ui
# NMSW-385 Change your details skeleton page

# Ticket

NMSW-385

----

## AC

No functionality but can put together page layout & render tests

----

## To test

- Run app
- Go to localhost:3000
- Sign in
- Click on Your details in the nav
- The design should match the link to the prototype in the ticket (new design since NMSW-384) 
- Click on 'Change your details'
- > You should be taken to a page with a blank form to 'change' your details (no pre-filling with user details yet)
- > It should match the prototype (but without the autocomplete fields for international dialling code and country)
- Click continue
- > All the fields should error
- Fill in dummy data
- Click continue
- > You should be taken to a generic confirmation page

----

## Notes
